src/test/fuzz/banman.cpp

```diff
@@ -93,6 +101,16 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
                     ban_man.Discourage(ConsumeNetAddr(fuzzed_data_provider));
                 });
         }
+        if (!force_read_and_write_to_err) {
+            ban_man.DumpBanlist();
+            SetMockTime(ConsumeTime(fuzzed_data_provider));
+            banmap_t banmap;
+            ban_man.GetBanned(banmap);
+            BanMan ban_man_read{banlist_file, /* client_interface */ nullptr, /* default_ban_time */ 0};
+            banmap_t banmap_read;
+            ban_man_read.GetBanned(banmap_read);
+            assert(banmap == banmap_read);
```

Comment:
Errors in CI job (https://cirrus-ci.com/task/6055643191705600?logs=ci#L2769):
```
fuzz: test/fuzz/banman.cpp:112: void banman_fuzz_target(FuzzBufferType): Assertion `banmap == banmap_read' failed.
```

Reply:
Is there a way to retrieve `./crash-0671aac15e619e99522e2119487eaa9cc97e5a34` from the CI machine or make the fuzzer print it in base64?

Latest `master` (d67330d11245b11fbdd5e2dd5343ee451186931e) has been changed to:

https://github.com/bitcoin/bitcoin/blob/d67330d11245b11fbdd5e2dd5343ee451186931e/src/test/fuzz/banman.cpp#L111-L113